platform/android/os_android.h

```diff
@@ -154,6 +155,8 @@ class OS_Android : public OS_Unix {
 	VideoPauseFunc video_pause_func;
 	VideoStopFunc video_stop_func;
 	SetKeepScreenOnFunc set_keep_screen_on_func;
+	
+	power_android power_manager;
```

Comment:
Should be `power_android *power_manager;`, right? This is what breaks the build for Android.